### PR TITLE
Refactor github actions

### PR DESCRIPTION
**Pull request by VincentLanglet**
- Use 4 spaces for indent
- Fix the failing phpcs tests

### CODE REVIEW

**Comment by VincentLanglet** [comment]
I'm not sure why the 7.4 test is not running and why I still have the phpcs pull_request_target test.
Does it require to be merged in order to see if it works ?

File: .github/workflows/auto-merge.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    pull_request:
+
+jobs:
+    auto-merge:
+        name: Auto-Merge
+
+        runs-on: ubuntu-latest
+
+        needs:
+            - test
```
Comment (OskarStark) [inline .github/workflows/auto-merge.yaml:16]: This does not work, it only knows about jobs in the same workflow

**Comment by OskarStark** [comment]
If it comes from a fork yes

File: .github/workflows/auto-merge.yaml (hunk 1 above)
Comment (VincentLanglet) [reply .github/workflows/auto-merge.yaml]: Didn't know this, I rollback the split.

**Review by OskarStark — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Pull request merged**